src/commands/games/2048.ts

```diff
@@ -305,7 +307,7 @@ export class TwentyFortyEight extends Command {
         });
 
         bm.on("timeout", async () => {
-            bm.btns.map((btn) => btn.setDisabled(true));
+            bm.btns.flat().map((btn) => btn.setDisabled(true));
```

Comment:
not that it matters but there's [`Array.flatMap()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap)

Reply:
nvm, flatMap maps first, then flattens